setup.cfg

```diff
@@ -22,12 +22,11 @@ packages = find:
 install_requires =
     click==8.1
     click-spinner==0.1.10
-    docker==6.1.2
+    docker==7.0.0
     packaging==21.3
-    python-dotenv==0.19.1
```

Comment:
This doesn't seem to be used anywhere, we're using our own custom solution to store configuration for profiles

Reply:
Yep, agree.